selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/git/GitCompare.java

```diff
@@ -56,6 +64,8 @@ public GitCompare(
     String LEFT_COMPARE_STATUS =
         "//div[@id='gwt-debug-compareParentDiv_left_status_id' and text()='%s']";
     String RIGHT_COMPARE_EDITOR = "gwt-debug-compareParentDiv_right_editor_id";
+    String NEXT_DIFF_BUTTON = "git-compare-next-diff-btn";
```

Comment:
Is it name or id? It's better to express nature of locator in the constant name by adding the prefix `_ID` or `_NAME`